coriolis/tests/conductor/rpc/test_server.py

```diff
@@ -2253,7 +2285,7 @@ def test_deploy_transfer_instances(
                 trust_id=None)
 
         self.assertEqual(mock_get_deployment.return_value, result)
-        self.assertEqual(mock.sentinel.user_scripts, deployment.user_scripts)
+        self.assertEqual({'instances': {}}, deployment.user_scripts)
```

Comment:
Below, please also add the following:
`mock_normalize_user_scripts.assert_called_once_with(mock.sentinel.user_scripts, transfer_mock.instances)`